### ISSUE TITLE
Sidebar Scrolling

### ISSUE THREAD

**Issue by TheGiraffe3** [Libera-Chat/libera-chat.github.io#362]
For some reason, the sidebar does not scroll. For me, if all drop-down menus are open, I can’t see the whole sidebar, and I can’t scroll it.

**Comment by xfnw** [comment]
possibly a duplicate of #39

**Comment by TheGiraffe3** [comment]
No, because for me it doesn’t scroll _at all_. #39 is to do it scrolling separately.

**Comment by swantzter** [comment]
Could you provide some more info, what browser and OS are you using?
